_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,329 @@
+share some insights with you all.
+
+Firstly - **thank you** to all those who filled out the survey. It was a little
+bit longer this year, as we looked to get data on how we can grow the
+contributor community as well as the user community, so thanks for taking the
+time. It's great to hear all your feedback.
+
+<!--more-->
+
+## <a name="intro"></a>Contents
+
+* [Intro](/2016/03/2016-forman-survey-analysis.html/#intro)
+* [Community Diversity](/2016/03/2016-forman-survey-analysis.html/#diversity)
+* [Support and Docs](/2016/03/2016-forman-survey-analysis.html/#support)
+* [Specific Features](/2016/03/2016-forman-survey-analysis.html/#foreman-features)
+* [Upcoming Work](/2016/03/2016-forman-survey-analysis.html/#upcoming-work)
+* [Events](/2016/03/2016-forman-survey-analysis.html/#events)
+* [Contributor Survey](/2016/03/2016-forman-survey-analysis.html/#contributor-survey)
+* [Final thoughts](/2016/03/2016-forman-survey-analysis.html/#final-thoughts)
+
+Sadly, response were slightly down on last year (**76 responses**) and the
+smaller sample size makes it difficult to draw strong conculsions. According to
+other metrics, we have usually 250ish active people in IRC each month, and some
+2000 people on the mailing list. Suggestions on how to reach more people with
+the survey are, of course, welcome!
+
+That said, we can still look at the results and look for trends, especially as
+compared to last year (which we couldn't do then, it being the first such
+survey). Lets start with some stats on...
+
+## <a name="diversity"></a>Community Diversity
+
+![How long using](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/how-long-using.png)
+
+This is broadly similar to last year, with some 40% of the community having
+used Foreman for under a year. We do see a shift the 3+ years category, with a
+corresponding drop in 1 and 2 year category, but it's nice to see that the 3
+months category remains stable at 13% - we seem to be continuing to attrach new
+people to the community, which is great. Having more "old hands" around to
+answer questions is starting to be noticable on IRC and the mailing lists, too.
+
+![What version used](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/what-version-using.png)
+
+I was quite surprised by this - Foreman 1.10 was released at Christmas, meaning
+it had only been out for 4 weeks by the time of the survey. It's great to see
+people so quick to upgrade something which can be so deeply embedded in the
+infrastructure. When you add in 1.9 (we still support the last two versions, so
+this is the total supported userbase) we get 71% which is a bit of a drop on
+last year (86%). A quarter of responses are running an unsupported version of
+Foreman - as we said last year, if there's anything holding you back from
+upgrading, do get in touch!
+
+![Nodes managed](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/nodes-managed.png)
```

Comment:
Yeah I can't do much about that without downloading the data and graphing it myself, which will then make it look different to all the other graphs. Google's results page gives you zero options to edit the displayed graphs, as far as I can tell.